src/test/java/sdmExam/PositionShould.java

```diff
@@ -1,51 +1,55 @@
 package sdmExam;
 
-import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.DynamicTest;
+import org.junit.jupiter.api.TestFactory;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.Arguments;
+import org.junit.jupiter.params.provider.MethodSource;
+
+import java.util.Arrays;
+import java.util.Collection;
+import java.util.stream.Stream;
 
 import static org.junit.jupiter.api.Assertions.*;
 
 public class PositionShould {
-    @Test
-    public void beEqualToAnotherPositionWithSameCoordinates() {
-        Position firstPosition = new Position(3, 3);
-        Position secondPosition = new Position(3, 3);
+
+    @ParameterizedTest
+    @MethodSource("providePosition")
+    public void beEqualToAnotherPositionWithSameCoordinates(Position firstPosition, Position secondPosition) {
         assertEquals(firstPosition, secondPosition);
     }
 
-    @Test
-    public void stoneAbove() {
-        Position firstPosition = new Position(1, 2);
-        Position secondPosition = new Position(2, 2);
-        assertTrue(firstPosition.isAboveWithRespectTo(secondPosition));
-    }
+    private static Stream<Arguments> providePosition(){
+        return Stream.of(
+                Arguments.of(Position.in(3,3),Position.in(3,3)),
+                Arguments.of(Position.in(13,13),Position.in(13,13)),
+                Arguments.of(Position.in(1,1),Position.in(1,1))
+        );
 
-    @Test
-    public void stoneBelow() {
-        Position firstPosition = new Position(3, 4);
-        Position secondPosition = new Position(2, 2);
-        assertFalse(firstPosition.isBelowWithRespectTo(secondPosition));
     }
 
-    @Test
-    public void stoneLeft() {
-        Position firstPosition = new Position(3, 4);
-        Position secondPosition = new Position(3, 5);
-        assertTrue(firstPosition.isOnTheLeftWithRespectTo(secondPosition));
-    }
-
-    @Test
-    public void stoneUpRight() {
-        Position firstPosition = new Position(5, 6);
-        Position secondPosition = new Position(6, 5);
-        assertTrue(firstPosition.isUpRightRespectTo(secondPosition));
-    }
+    @TestFactory
+    Collection<DynamicTest> checkPositions() {
+        Position firstPosition = new Position(1, 2);
```

Comment:
Use the factory method `in` instead of the constructor. Just to be more consistent, in some parts you use the factory method, in others, you use the constructor, so to make it more clear, use just the factory method.

Reply:
Done